summingbird-storm/src/main/scala/com/twitter/summingbird/storm/StormPlatform.scala

```diff
@@ -24,6 +24,7 @@ import backtype.storm.spout.SpoutOutputCollector
 import backtype.storm.topology.{ BoltDeclarer, IRichBolt, IRichSpout, OutputFieldsDeclarer, TopologyBuilder }
 import backtype.storm.tuple.{ Fields, Tuple }
 import backtype.storm.{ LocalCluster, StormSubmitter, Config => BacktypeStormConfig }
+import com.twitter.algebird.util.summer.Incrementor
```

Comment:
With the `SpoutProvider` refactoring, you can probably clean up the imports here quite a bit.  At the very least, drop any of the changes which aren't actually needed.